neuralmonkey/nn/projection.py

```diff
@@ -19,9 +19,12 @@ def linear(inputs, size, scope="LinearProjection"):
         A tensor of shape batch x size
     """
     with tf.variable_scope(scope):
-        return tf.nn.seq2seq.linear(inputs, size, True)
+        # TODO this will be soon deprecated, should be rewrited with
+        # tf.contrib.layers.fully_connected and a support for list of tensors
+        return tf.nn.seq2seq.linear(inputs, size, True, scope=scope)
 
 
+# SUGGESTION: what about using ReLu as a default? It have better results
```

Comment:
Ja si myslim, ze stejne to nepujde poradne 100% schopne reprodukovat a pak naopak je lepsi, kdyz tam bude neco, co se chova lepe nez horsi default. Ale mozna to je muj nazor, kdy uprednostnuju lepsi vysledky nad porovnatelnosti s jinyma frameworkama. Reseni by asi bylo nedavat tam zadny default, ale to podle me rozbije dost veci.